index.py

```diff
@@ -7,12 +7,12 @@
 
 app = Flask(__name__)
 
-app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX', '/applist')
+app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX')
```

Comment:
Should it fail somewhere if that's `None`?

Reply:
Same for the other configured variables below.

Reply:
@briehl @jayrbolton What should the error message be for `if conf['DASHBOARD_ENDPOINT'] = None:`??

Reply:
I'd do something like:
```
assert os.environ.get('DASHBOARD_ENDPOINT', '').strip(), "DASHBOARD_ENDPOINT env var is required."
```